Control/DeepSeq.hs

```diff
@@ -355,7 +406,11 @@ instance NFData (a -> b) where rnf = rwhnf
 
```

Comment:
This might be a contentious point (given the existence of #16), but there _could_ be `NFData1 ((->) a)` and `NFData2 (->)` instances.